doc/tutorials/basic/index.md

```diff
@@ -0,0 +1,332 @@
+# Basic Tutorials
```

Comment:
Please as many as possible review this into section (lines 1-9). This defines and scopes what Basic Tutorials are. 

If we don't share a common understanding and agreement we can't really discuss details below in a shared context. @philippjfr